### ISSUE TITLE
Excel steals focus and keyboard input from the Script Lab pane

### ISSUE THREAD

**Issue by Narvey** [OfficeDev/script-lab#502]
## Expected Behavior ##

When I click or select text in the code editor, any subsequent keystrokes should be sent to the code editor, of course.

## Current Behavior ## 

Sometimes, (and various things will start/stop this behavior on an inconsistent basis, including alt-tab), the keystrokes (including Delete, yikes!) apply to the excel spreadsheet, and goof up the data in my spreadsheet cells, even when it looks like focus is in the code editor. I've a hunch this is related to the other focus issues in #383, but it manifests itself differently.

## Context ##
I was simply trying to type in the ScriptLab code editor.

## Your Environment ##
* Version used: 1.1.0
* Excel 2016 MSO (16.0.4498.1000) (hey that's __weird__, I had to use the old Office 2013 Addins method, why don't I have ribbon buttons for Code and Run!?)
* Operating System and version (desktop or mobile): Windows 10 Enterprise (Desktop)

**Comment by changguangyu** [comment]
Same question

**Comment by Zlatkovsky** [comment]
@saunders77 , do you know if we have an in-house repro/understanding of what is happening?  It seems like an underlying platform issue, not Script-Lab specific...

**Comment by Narvey** [comment]
@Zlatkovsky, yeah, especially since something seems fishy about my install (it reports itself as 2016 MSO, but it doesn't have ribbom buttons for Code and Run like Excel 2016 is supposed to).
